packages/react-tinacms-strapi/src/tina-strapi-client.ts

```diff
@@ -0,0 +1,11 @@
+export const STRAPI_JWT = 'tina_strapi_jwt'
+
+export class TinaStrapiClient {
+  async authenticate(username: string, password: string) {
+    return fetch(`${process.env.STRAPI_URL}/auth/local`, {
```

Comment:
- [ ] Accept `STRAPI_URL` as a constructor argument so this package doesn't rely on a specific environment variable